canonical-web-and-design/ubuntu.com
# WIP add ubuntu pro ebook page

## Done

**BLOCKED:** we're waiting on the ebook asset

- Added /azure/ebook according to the [design](https://github.com/canonical-web-and-design/web-squad/issues/4809#issuecomment-1040139905)
  - this page was [originally part of a PR](https://github.com/canonical-web-and-design/web-squad/issues/4823) that included another, related page, but we're still waiting on the ebook asset, so I closed that PR and created two separate PRs instead. I've added the Design +1 label, as it was given there, and the design hasn't changed in the process of creating this PR.

## QA

- View the site locally in your web browser at: https://ubuntu-com-11285.demos.haus/azure/ebook
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the site matches the copy doc *(the copy doc doesn't exist yet, I'll create it before asking for a review on this)*

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4824
